doc/observability-deployment.adoc

```diff
@@ -115,7 +121,7 @@ spec:
 === Visualizing your data with Grafana
 
 
-There are IBM provided Grafana dashboards that leverage metrics from the JVM as well as from the Open Liberty runtime.  If you are deploying Grafana on Openshift, see the following link:openshift-monitoring.adoc[guide].
+There are IBM provided Grafana dashboards that leverage metrics from the JVM as well as from the Open Liberty runtime.  If you are deploying Grafana on OpenShift, see the following link:openshift-monitoring.adoc[guide].
 
 Sample Open Liberty Grafana dashboards are available for servers using either mpMetrics-1.x or mpMetrics-2.x link:++https://github.com/OpenLiberty/open-liberty-operator/tree/main/deploy/dashboards/metrics++[here]. Look in the featureManager section of the server.xml for either the mpMetrics feature or the umbrella microProfile feature to determine which dashboard to use.
 
```

Comment:
table below needs to be updated with newer versions of microProfile / mpMetrics.